src/pages/Login/Login.scss

```
@@ -1 +1,69 @@
 @import '../../styles/variables.scss';
+@import url('https://fonts.googleapis.com/css2?family=Tangerine&display=swap');
+
+.login {
+  background-color: #ffffff;
+  @include flexCenter;
```

include를 최상단에 부여해주세요!
```suggestion
  @include flexCenter;
  background-color: #ffffff;
```